pkg/agent/core/goalcu.go

```diff
@@ -65,11 +66,29 @@ func calculateCPUGoalCU(
 	computeUnit api.Resources,
 	systemMetrics SystemMetrics,
 ) uint32 {
-	goalCPUs := systemMetrics.LoadAverage1Min / *cfg.LoadAverageFractionTarget
+	stableThreshold := *cfg.CPUStableZoneRatio * systemMetrics.LoadAverage5Min
+	mixedThreshold := stableThreshold + *cfg.CPUMixedZoneRatio*systemMetrics.LoadAverage5Min
+
+	diff := math.Abs(systemMetrics.LoadAverage1Min - systemMetrics.LoadAverage5Min)
+	load1Weight := blendingFactor(diff, stableThreshold, mixedThreshold)
```

Comment:
```suggestion
	// load1Weight is 0 when diff < stableThreshold, and 1 when diff > mixedThreshold.
	// If diff is between the thresholds, it'll be between 0 and 1.
	load1Weight := blendingFactor(diff, stableThreshold, mixedThreshold)
```

I had a hard time figuring out what was happening here the first time I read it.

Wording isn't strict -- you probably have a better idea about how to explain mixedThreshold :sweat_smile: